src/com/cloudogu/ces/cesbuildlib/GradleInDockerBase.groovy

```diff
@@ -22,13 +23,26 @@ abstract class GradleInDockerBase extends Gradle {
     abstract def call(Closure closure, boolean printStdOut);
 
     protected void inDocker(String imageId, Closure closure) {
-        docker.image(imageId)
-        // Mount user and set HOME, which results in the workspace being user.home. Otherwise '?' might be the user.home.
+        if (Utility.isBlank(this.credentialsId)) {
```

Comment:
Is `trim()` so important here to justify a utility?
Most of the times theses checks are just `if (this.credentialsId)`